layouts/New/layouts.module.css

```diff
@@ -0,0 +1,64 @@
+.baseLayout {
+  @apply grid
+    h-screen
+    w-screen
+    grid-cols-[1fr]
+    grid-rows-[auto_1fr_auto];
+}
+
+.articleLayout {
+  @apply grid
+    min-h-full
+    min-w-full
```

Comment:
Why add min ? it's can be just full
```suggestion
    h-full
    w-full
```

Reply:
I don't think so. It has different effects.

Reply:
in theory yes, but in our case it doesn't make any difference

Reply:
Actually we don't even need these at all, because of grid.